src/PDFsharper.UnitTests/Pdf.AcroForms/PdfAcroFieldTests.cs

```diff
@@ -78,43 +78,7 @@ public void GetIsVisible_WithFlagsRemoved()
             Assert.IsFalse(isNotVisible, "isVisible should be false");
         }
 
-        [TestMethod]
-        public void RemoveJavascript()
```

Comment:
The javascript based error only happens when all fields are removed from the form. These methods were pulled for that reason and for performance, flattening with these in place was sluggish.